src/terminal/input/terminalInput.hpp

```diff
@@ -13,28 +13,32 @@ Author(s):
 - Michael Niksa (MiNiksa) 30-Oct-2015
 --*/
 
-#include <functional>
-#include "../../types/inc/IInputEvent.hpp"
 #pragma once
 
+#include "../../types/inc/IInputEvent.hpp"
+
 namespace Microsoft::Console::VirtualTerminal
 {
     class TerminalInput final
     {
     public:
-        TerminalInput(_In_ std::function<void(std::deque<std::unique_ptr<IInputEvent>>&)> pfn);
+        using StringType = std::wstring;
+        using OutputType = std::optional<StringType>;
 
-        TerminalInput() = delete;
-        TerminalInput(const TerminalInput& old) = default;
-        TerminalInput(TerminalInput&& moved) = default;
+        static_assert(StringType{} == StringType{});

```

Comment:
Why do we need this?

Reply:
Ah woops that's a leftover from when I was testing which `StringType` would work with the unit tests. I needed a class that as a `==` operator. I'll remove it. 😅